docs/app/pages/components/components-sections/media-player/media-player/media-player.component.html

```diff
@@ -23,6 +22,13 @@
                         <ux-radio-button [(value)]="mode" option="quiet" [disabled]="type === 'audio'">Quiet</ux-radio-button>
                     </div>
                 </div>
+
+                <div class="row uxd-customize-row">
```

Comment:
I'm not sure this requires an customize input as by default it will display the filename. This is only really relevant if we are using a base64 file as the `src`.

Reply:
Yes, there should be no change to the documentation example for this ticket, since it uses a named file.